src/common/transformations/src/transformations/common_optimizations/fuse_moe_expert.cpp

```diff
@@ -0,0 +1,340 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/fuse_moe_expert.hpp"
+
+#include <cstdint>
+#include <limits>
+
+#include "itt.hpp"
+#include "openvino/core/graph_util.hpp"
+#include "openvino/core/rt_info.hpp"
+#include "openvino/op/util/shape_of_base.hpp"
+#include "openvino/opsets/opset1.hpp"
+#include "openvino/opsets/opset11.hpp"
+#include "openvino/opsets/opset12.hpp"
+#include "openvino/opsets/opset13.hpp"
+#include "openvino/opsets/opset15.hpp"
```

Comment:
usually we don't use opset header in the transformation code (win debug build issue, the lib exceed 4gb in debug mode)
please use concrete op versions instead

for example
opset1::Add -> ov::v1::Add

please use the latest version of operations

Reply:
Done.